bapsflib/_hdf/maps/controls/nixyz.py

```diff
@@ -0,0 +1,206 @@
+# This file is part of the bapsflib package, a Python toolkit for the
+# BaPSF group at UCLA.
+#
+# http://plasma.physics.ucla.edu/
+#
+# Copyright 2017-2019 Erik T. Everson and contributors
+#
+# License: Standard 3-clause BSD; see "LICENSES/LICENSE.txt" for full
+#   license terms and contributor agreement.
+#
+import h5py
+import numpy as np
+
+from bapsflib.utils.errors import HDFMappingError
+from warnings import warn
+
+from .contype import ConType
+from .templates import HDFMapControlTemplate
+
+
+class HDFMapControlNIXYZ(HDFMapControlTemplate):
+    """
+    Mapping module for control device 'NI_XYZ'.
+
+    Simple group structure looks like:
+
+    .. code-block:: none
+
+        +-- NI_XYZ
+        |   +-- <motion list name 1>
+        |   |   +--
+        .
+        .
+        .
+        |   +-- <motion list name N>
+        |   |   +--
+        |   +-- Run time list
+    """
+    def __init__(self, group: h5py.Group):
+        """
+        :param group: the HDF5 control device group
+        """
+        HDFMapControlTemplate.__init__(self, group)
+
+        # define control type
+        self._info['contype'] = ConType.motion
+
+        # populate self.configs
+        self._build_configs()
+
+    def _build_configs(self):
+        """Build the :attr:`configs` dictionary"""
+        # Assumptions:
+        # 1. only one NI_XYZ drive was ever built, so there will always
+        #    be only one configuration
+        #    - naming configuration 'config01'
+        # 2. there's only one dataset ever created 'Run time list'
+        # 3. there can be multiple motion lists defined
+        #    - each sub-group is a configuration for a different
+        #      motion list
+        #    - the name of the sub-group is the name of the motion list
+        #
+        # initialize configuration
+        cname = 'config01'
+        self.configs[cname] = {}
+
+        # check there are existing motion lists
+        if len(self.subgroup_names) == 0:
+            warn(self.info['group path']
+                 + ': no defining motion list groups exist')
+
+        # get dataset
+        try:
+            dset = self.group[self.construct_dataset_name()]
+        except KeyError:
+            why = ("Dataset '" + self.construct_dataset_name()
+                   + "' not found")
+            raise HDFMappingError(self.info['group path'], why=why)
+
+        # ---- define general config values                         ----
+        # none exist
```

Comment:
add general items about probe drive pivot (see [comment](https://github.com/BaPSF/bapsflib/pull/39#issuecomment-492894317))

```suggestion
        self.configs[cname].update({
            'Note': "The 'r', 'theta', and 'phi' fields in the "
                    "NI_XYZ data set are suppose to represent "
                    "spherical coordinates of the probe tip with "
                    "respect to the pivot point of the probe drive, "
                    "but the current calculation and population of the"
                    "fields is inaccurate.  For user reference, the "
                    "distance between the probe drive pivot point and"
                    "LaPD axis is (Lpp =) 58.771 cm.",
            'Lpp': 58.771 * u.cm,
        })
```